classes/Policy.php

```diff
@@ -92,15 +93,15 @@ public function foreignKey(string $class): string
     /**
      * Get properties for class
      *
-     * @param string $class
+     * @param string|object $class
      * @return Property[]
      */
-    public function properties(string $class)
+    public function properties($class)
```

Comment:
BWC break needs documentation or reevaluation; any Policy subclass needs to accommodate for object arguments after this change, is this necessary?

Reply:
This is not necessary however it doesn't require any additional changes in our implementation and allows nicer syntax in consuming code.
For example in a relation builder the following code could be written in favor of the second line without the change
```php
$this->property($this, 'id');
$this->property(get_class($this), 'id');
```